src/main/java/entity_layer/CommonTimeBlock.java

```diff
@@ -11,6 +12,26 @@
  */
 public class CommonTimeBlock implements TimeBlock {
 
+    /* ********** *\
+    *  Attributes  *
+    \* ********** */
+    private LocalDateTime startTime;
+    private LocalDateTime endTime;
+
+    /* ************ *\
+    *  Constructors  *
+    \* ************ */
+    public CommonTimeBlock(LocalDateTime startTime, LocalDateTime endTime) {
+        this.startTime = startTime;
+        this.endTime = endTime;
+        // I'll have to include a check somewhere that doesn't allow start time to be before
```

Comment:
nonsensical comment

Reply:
If you are going to implement the check do so, if not, let a higher level object implement it